unattended-upgrade

```diff
@@ -87,6 +87,7 @@ import distro_info
 
 # the reboot required flag file used by packages
 REBOOT_REQUIRED_FILE = "/var/run/reboot-required"
+KEPT_PACKAGES_FILE = "var/lib/unattended-upgrades/kept-back"
```

Comment:
Isn't it missing a leading / ?

Reply:
No, I'm transitioning to prepend rootdir to global paths an it marks that.